### ISSUE TITLE
Can't Install RoundCube because of PEAR requirements

### ISSUE THREAD

**Issue by nickian** [mailcow/mailcow-dockerized#3566]
Just following the directions listed here: https://mailcow.github.io/mailcow-dockerized-docs/third_party-roundcube

However, on the RoundCube installer, I get the errors below. I'm trying to install 1.4.4. Are there only specific versions that work, or am I missing something? Thanks.

```
Auth_SASL:  NOT OK(Failed to load class Auth_SASL from pear.php.net)

Net_SMTP:  NOT OK(Failed to load class Net_SMTP from pear.php.net; See http://pear.php.net/package/Net_SMTP)

Net_IDNA2:  NOT OK(Failed to load class Net_IDNA2 from pear.php.net; See http://pear.php.net/package/Net_IDNA2)

Mail_mime:  NOT OK(Failed to load class Mail_mime from pear.php.net; See http://pear.php.net/package/Mail_mime)
```

<img width="1707" alt="Screen Shot 2020-05-23 at 2 10 21 AM" src="https://user-images.githubusercontent.com/590678/82725495-99f3a680-9c9a-11ea-9afb-6b62fc102158.png">

**Comment by andryyy** [comment]
You need to follow the docs. Do exactly what you see there. 
You are not using the complete package. 

I copy/pasted the instructions just yesterday.

**Comment by nickian** [comment]
I solved it, but there were a couple of things to note that could be helpful to others:

1. Make sure to grab `roundcubemail-1.4.4-complete.tar.gz ` and NOT `roundcubemail-1.4.4.tar.gz`

2. I had to set `$config['enable_installer'] = true` in `defaults.inc.php` as well as `config.inc.php`

3. I had to do this step BEFORE completing the installer and deleting it:

Open `data/web/rc/plugins/password/config.inc.php` and change the following parameters (or add them at the bottom of that file):

```
$config['password_driver'] = 'sql';
$config['password_algorithm'] = 'ssha256';
$config['password_algorithm_prefix'] = '{SSHA256}';
$config['password_query'] = "UPDATE mailbox SET password = %P WHERE username = %u";
```
